gslib/tests/util.py

```diff
@@ -543,19 +598,30 @@ def GetTestNames():
       names.append(m.group('name'))
   return names
 
+def is_file(obj):
+  if six.PY2:
+    return isinstance(obj, file)
+  else:
```

Comment:
Nit: This `else:` is unnecessary, the code will still get executed if we fall through the if check above.

Reply:
Removed `else` from method